service_thing_aggregate/src/main/scala/controllers/ObjectSearchController.scala

```diff
@@ -0,0 +1,49 @@
+package controllers
+
+import com.google.inject.Inject
+import models.MusitThing
+import no.uio.musit.service.MusitResults.{MusitDbError, MusitError, MusitResult, MusitSuccess}
+import play.api.Logger
+import play.api.libs.concurrent.Execution.Implicits.defaultContext
+import play.api.libs.json.{JsValue, Json}
+import play.api.mvc.{Action, Controller, Result, Results}
+import services.ObjectSearchService
+
+/**
+ * Created by jarle on 05.10.16.
+ */
+
+object Utils {
```

Comment:
I had it separately because I believe that something like this ought to be in some common file. To go from MusitResult to (Play)Result is something we have to do in all end points. It's too easy to screw this up, so we should have a standard way of handling it.